tracer/build/_build/NukeExtensions/DotNetSettingsExtensions.cs

```diff
@@ -197,20 +197,25 @@ public static DotNetTestSettings WithMemoryDumpAfter(this DotNetTestSettings set
         );
     }
 
-    public static DotNetTestSettings WithDatadogLogger(this DotNetTestSettings settings)
+    public static DotNetTestSettings WithDatadogLogger(this DotNetTestSettings settings, bool isLocalBuild)
```

Comment:
This is such a small PR that I feel too nitpicky here, but it makes more sense in my brain for this parameter to be named `defaultValue`, and then we could pass in either `!IsLocalBuild` or `IsServerBuild`. This would allow us to get rid of the ternary operator later and make this change even smaller. WDYT?

Reply:
Also, it took me wayyy too long to realize that `IsLocalBuild` is a [Nuke property](https://nuke.build/docs/fundamentals/builds/). I couldn't find it anywhere in our code and was confused how this was working! 😆

Reply:
Thanks, I agree, though I went one better I think - I forgot the `IsLocalBuild`/`IsServerBuild` flags are static, so there's no need to pass in the value at all, we can grab it from the extension. And then simplified the logic a bit